package.json

```diff
@@ -114,17 +113,17 @@
   },
   "devDependencies": {
     "@actions/core": "^1.2.6",
-    "@actions/github": "^4.0.0",
+    "@actions/github": "^5.0.3",
     "@babel/core": "^7.11.1",
     "@babel/plugin-proposal-class-properties": "^7.12.1",
     "@babel/preset-env": "^7.11.0",
     "@babel/preset-flow": "^7.12.13",
     "@babel/preset-react": "^7.10.4",
     "@babel/runtime": "^7.11.2",
-    "@octokit/core": "^3.3.1",
```

Comment:
Need to look into this change further

Reply:
Doesn't look like this introduced any breaking changes https://github.com/octokit/core.js/releases/tag/v4.0.0

Reply:
Actually, I was wrong, this introduces a breaking change I believe.

Reply:
Fixed in b7643e4